### DISCUSSION TITLE
Welcome to .github Discussions!

### DISCUSSION THREAD

**Discussion by rndmh3ro** [dev-sec/.github#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by deric4** [comment]
oh. this looks like discussions is only enabled for this project and not for the organization?

**Reply by rndmh3ro** [reply]
My bad. It should work now as expected.
